app/keepers/modules.go

```diff
@@ -28,6 +28,7 @@ import (
 	ica "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts"
 
 	_ "github.com/osmosis-labs/osmosis/v13/client/docs/statik"
+	downtimemodule "github.com/osmosis-labs/osmosis/v13/x/downtime-detector/downtimedetector_module"
```

Comment:
nit: `downtime-detector/downtimedetector_module` is ugly. Can it be `downtime-detector/module`? or `sdkmodule`, or something similar?